crates/store/src/db/migrations/2025062000000_setup/up.sql

```diff
@@ -91,13 +91,12 @@ CREATE TABLE note_scripts (
 CREATE TABLE account_storage_map_values (
     account_id          BLOB NOT NULL,
     block_num           INTEGER NOT NULL,
-    slot                INTEGER NOT NULL,
+    slot_name           BLOB NOT NULL,
```

Comment:
Related to one of the previous comments: I wonder if we should store both `slot_name` and `slot_id` here - primarily to make querying faster (the assumption here is that querying by `slot_id` would be faster than by `slot_name` - though, if there is a meaningful difference would need to be validated).